istioctl/cmd/injector-list.go

```diff
@@ -303,6 +303,9 @@ func hideFromOutput(ns resource.Namespace) bool {
 }
 
 func injectionDisabled(pod *v1.Pod) bool {
+	if pod.ObjectMeta.GetAnnotations() == nil{
```

Comment:
why do we need this? A nil map is valid

Reply:
I thought something would go wrong with the null pointer, but it seems I was wrong。🤣